test/test.js

```diff
@@ -162,6 +162,25 @@
         });
         checkApi('regexp');
 
+        describe('is.map', function() {
+            it('should return true if passed parameter type is map', function() {
+                var ctx = (typeof window === 'undefined') ? global : window;
```

Comment:
Would you move `ctx` to the top of the test file?
I see the `ctx` juggle as something that other tests could use.